h2o-py/h2o/explanation/_explain.py

```diff
@@ -951,33 +951,100 @@ def mapping(x):
         plt.xticks(mapping(range(nf.nlevels(column))), nf.levels(column))
     plt.ylim(ylims)
 
-def _handle_ice(model, frame, colormap, plt, target, is_factor, column, show_logodds, centered, factor_map, show_pdp, **kwargs):
+
+def _append_graphing_data(graphing_data, data_to_append, original_observation_value, frame_id, centered, show_logoods,
+                          row_id, **kwargs):
+    grouping_variable_value = kwargs.get("grouping_variable_value", None)
```

Comment:
nit/optional: No need to return `None` as default, as it's default anyways.

Reply:
[f2bd46e](https://github.com/h2oai/h2o-3/pull/6119/commits/f2bd46e90cf03fc39da9cffae0cbe66a7064279c)